content/manuals/compose/bridge/usage.md

```diff
@@ -22,19 +24,38 @@ Compose Bridge supplies an out-of-the-box transformation for your Compose config
 It also supplies a Kustomize overlay dedicated to Docker Desktop with:
  - `Loadbalancer` for services which need to expose ports on host.
  - A `PersistentVolumeClaim` to use the Docker Desktop storage provisioner `desktop-storage-provisioner` to handle volume provisioning more effectively.
- - A Kustomize file to link all the resources together.
+ - A `Kustomization.yaml` file to link all the resources together.
+
+If your Compose file defines a `models` section for a service, Compose Bridge automatically configures your deployment so your service can locate and use its models via Docker Model Runner.
+
+For each declared model, the transformation injects two environment variables:
+
+- `<MODELNAME>_URL`: The endpoint for Docker Model Runner serving that model  
+- `<MODELNAME>_MODEL`: The model’s name or identifier
+
+You can optionally customize these variable names using `endpoint_var` and `model_var`.
+
+The default transformation automatically detects which environment you’re targeting and automatically generates the necessary manifests. 
+
+| Environment    | Endpoint                                        |
+| -------------- | ----------------------------------------------- |
+| Docker Desktop | `http://host.docker.internal:12434/engines/v1/` |
+| Kubernetes     | `http://model-runner/engines/v1/`               |
+
+
+For more details, see [Use Model Runner](use-model-runner.md).
 
 ## Use the default Compose Bridge transformation
 
-To use the default transformation run the following command:
+To convert your Compose file using the default transformation:
 
 ```console
 $ docker compose bridge convert
 ```
 
-Compose looks for a `compose.yaml` file inside the current directory and then converts it.
+Compose looks for a `compose.yaml` file inside the current directory and generates Kubernetes manifests.
 
-When successful, Compose Bridge generates Kubernetes manifests and logs output similar to the following:
+Example output:
 
 ```console
 $ docker compose bridge convert -f compose.yaml 
```

Comment:
I can provide you an output with the Docker Model kubernetes resources if you want

Reply:
yes please @glours